aspnetcore/fundamentals/app-state.md

```diff
@@ -16,28 +16,26 @@ ms.custom: H1Hack27Feb2017
 
 # Introduction to session and application state in ASP.NET Core
 
-By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Steve Smith](http://ardalis.com)
+By [Rick Anderson](https://twitter.com/RickAndMSFT), [Steve Smith](http://ardalis.com)  and [Diana LaRose](https://github.com/DianaLaRose)
```

Comment:
Sorry ... I should've checked the commit history.

Rick did that! ... He's testing me. :smile: He wants to make sure I'm still on the ⚽️.